argus/backend/util/config.py

```diff
@@ -12,6 +12,7 @@ class Config:
     CONFIG_PATHS = [
         Path("./config/argus_web.yaml"),
         Path("argus_web.yaml"),
+        Path("../config/argus_web.yaml"),
```

Comment:
wouldn't it be easy to just usg globbing of something like that ?

Reply:
I didn't think much here, just added a line when testing.
Your'e right, this could be done better - I'll do it later today.